docs/config.md

```diff
@@ -212,7 +212,7 @@ are used to mitigate assembly version conflicts in .NET Core.
 
 | Environment variable | Required value |
 |-|-|
-| `DOTNET_STARTUP_HOOKS` | `%InstallationLocation%/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll` |
-| `DOTNET_ADDITIONAL_DEPS` | `%InstallationLocation%/AdditionalDeps` |
-| `DOTNET_SHARED_STORE` | `%InstallationLocation%/store` |
+| `DOTNET_STARTUP_HOOKS` | `$INSTALL_DIR/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll` |
```

Comment:
Bummer, it will get verbose when adding Windows syntax... perhaps resolve all of these with a note on a central place? E.g.: "using unix env var syntax on the docs unless otherwise noted blah blah"

@theletterf any suggestions on how to handle the diff syntaxes and keep it short? Or do you think it is better to go with unix and Windows syntaxes?

Reply:
Actually, doesn't PowerShell support the dollar sign as a var mark? Do we have to wrap the statements in double quotes to make it safe? 

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_variables?view=powershell-7.2#using-variables-in-commands-and-expressions

If those can be used for both Bash and PSh, I'd rename the column heading "Requires value" to "Required value (Bash and Powershell)

Reply:
PowerShell would be `$Env:ENV_VAR_NAME` - the worst `$ENV_VAR_NAME` is treated as a variable and since it was not defined empty ...

That said I'm fine with using PowerShell syntax as long as it is something that works on Windows without a bash shell 😬